python/orca/example/torchmodel/train/resnet_finetune/resnet_finetune.py

```diff
@@ -47,29 +49,28 @@ def forward(self, x):
 
 if __name__ == '__main__':
 
-    if len(sys.argv) != 2:
+    if len(sys.argv) == 1:
```

Comment:
delete this if. Because you have add a OptionParser

Reply:
Updated in new commit.